web/src/components/Contact.jsx

```diff
@@ -0,0 +1,39 @@
+"use client";
+import Image from "next/image";
+import React from "react";
+import { HiOutlineMail } from "react-icons/hi";
+
+const About = () => {
+  return (
+    <div className="hero min-h-screen bg-alpha">
+    <div className="hero-content flex-col lg:flex-row-reverse">
+      <div className="text-center lg:text-left">
+        <h1 className="text-5xl font-bold">Contact ✨</h1>
+        <p className="py-6 text-3xl">Keep in touch with us to hear happy stories and nice monent everytime! <br/><span className="italic font-bold">Cows do love you!</span></p>
+    </div>
+    <div className="card shrink-0 w-full max-w-3xl shadow-2xl bg-beta">
```

Comment:
# Review to fix
Change emoji in Contact.jsx in line 11 to fit the context. Please change ✨ into something meaningful like 🌼 or 🐄. Because our team is mainly about cows.